rest-openehr/src/main/java/org/ehrbase/rest/openehr/OpenehrTemplateController.java

```diff
@@ -165,13 +164,11 @@ public ResponseEntity<StructuredString> getTemplateExample(
             respHeaders.setContentType(MediaType.APPLICATION_JSON);
```

Comment:
should this not change depending on format?

Reply:
Have a look at the preceding lines :)